czech.h

```diff
@@ -243,6 +243,11 @@
 #define řeťdondd(x, y, z) strtoull(x, y, z)
 
 #define náho() rand()
+// semeno náhody
+#define semnáho(x) srand(x)
```

Comment:
přidal bych i alternativu `snáho`

Reply:
nahradil jsem to úplně, aby to lépe odpovídalo `srand`